.gitignore

```diff
@@ -34,3 +34,5 @@ node_modules
 # Local dev
 Dockerfile
 docker-compose.yml
+
+tests/migrations/*
```

Comment:
since I don't define the BigAutoField in 1.10, it breaks the migration file. I *could* change that but I think it's not worth changing.